Issue 12

idovgalyov-4xxi opened the pull request:
Modified the code to make it more understandable.
Did some additional optimizations.
Updated function "find_asteroid".
Split the subroutine "mercury_processing" in several parts.
The unit tests are left the same (need more conversations).

idovgalyov-4xxi committed:
Issue-12 WIP-1

Updated function "find_asteroid"

idovgalyov-4xxi committed:
Issue 12

Split the procedure "mercury_processing" in several parts
Modified the code to make it more understandable.
Did some additional optimizations.

smirik reviewed (changes requested):
1. Try to split integrator.f90 into:
a) mercury_adapter.f90
b) astdys.f90

In this case, all logic related to mercury would be moved to mercury_adapter.f90 file. Astdys specific would be moved to separated file. And it should be OK.

2. Add tests to the new files & verify them with travis.

idovgalyov-4xxi committed:
Issue 12 (2)

Split module 'integrator' into 'astdys_adapter' and 'mercury_adapter' (two different directories).
Did some changes in files that are responsible for compiling and cleaning.
It would be easier here to take care about unit tests after all modifications of the program.
